test/fail_compilation/test13536.d

```diff
@@ -0,0 +1,21 @@
+/*
+---
+fail_compilation/test13536.d(19): Error: field U.safeDg cannot be accessed in @safe code because it overlaps with a pointer
```

Comment:
The other case was fixed and tested in another bug fix. This just adds the OPcall case.